crates/fj-export/src/lib.rs

```diff
@@ -48,4 +67,57 @@ pub fn export(mesh: &Mesh<Point<3>>, path: &Path) -> Result<(), Error> {
     Ok(())
 }
 
-pub use threemf::Error;
+fn export_stl(mesh: &Mesh<Point<3>>, path: &Path) -> Result<()> {
+    let points = mesh
+        .triangles()
+        .map(|triangle| triangle.points)
+        .collect::<Vec<_>>();
+
+    let vertices = points.iter().map(|points| {
+        points.map(|point| {
+            [point.x.into_f32(), point.y.into_f32(), point.z.into_f32()]
+        })
+    });
+
+    let normals = points
+        .iter()
+        .map(|&points| points.into())
+        .map(|triangle: Triangle<3>| triangle.to_parry().normal())
+        .collect::<Option<Vec<_>>>()
+        .ok_or_else(|| anyhow!("Unable to compute normal"))?;
+
+    let normals = normals.iter().map(|vector| vector.into_inner().into()).map(
+        |vector: Vector<3>| {
+            [
+                vector.x.into_f32(),
+                vector.y.into_f32(),
+                vector.z.into_f32(),
+            ]
```

Comment:
This kind of code can be shortened a bit:
```suggestion
            vector.components.map(|s| s.into_f32())
```

Not necessary, just noting the possibility.